package.json

```diff
@@ -2,7 +2,7 @@
   "name": "detect-adblock",
   "version": "0.0.1",
   "description": "Implements detecting of adblockers in browser",
-  "main": "lib/index.js",
+  "main": "src/index.js",
```

Comment:
This should not be changed because we expect to install it from `npm`. In case you are doing git clone, `yarn build`should be done before running.